android/src/com/mapswithme/maps/routing/NavigationController.java

```diff
@@ -59,7 +59,7 @@
 
   private final SearchWheel mSearchWheel;
 
-  private boolean mShowTimeLeft = true;
+  private static boolean mShowTimeLeft = true;
```

Comment:
I'm really concerned about this fix. If problem was appeared during the rotation screen then I would add additional methods in this class such as saveState/restorState and call it from MwmActivity which is owner of this controller. I think it would be much better solution then declaring the variable static. @fijemax  what do you think (maybe I'm wrong)?